packages/material-ui/src/OutlinedInput/NotchedOutline.js

```diff
@@ -5,63 +5,61 @@ import withStyles from '../styles/withStyles';
 import useTheme from '../styles/useTheme';
 import capitalize from '../utils/capitalize';
 
-export const styles = theme => {
-  return {
-    /* Styles applied to the root element. */
-    root: {
-      position: 'absolute',
-      bottom: 0,
-      right: 0,
-      top: -5,
-      left: 0,
-      margin: 0,
-      padding: 0,
-      paddingLeft: 8,
-      pointerEvents: 'none',
-      borderRadius: 'inherit',
-      borderStyle: 'solid',
-      borderWidth: 1,
+export const styles = theme => ({
```

Comment:
For callbacks you usually argue for an explicit return. Why the change of heart?

Reply:
Can we just leave it as is to have a clean git history? The implicit return is always bad for future change. I get that for the initial commit but why change it arbitrarily?

Reply:
The change is only motivated for consistency. For the better or the worth, it's the approach we use everywhere else. Is it better? I have no idea.

Reply:
> Is it better? I have no idea.

I just gave you one reason why it's not. Even if you ignore that you shouldn't arrive at the conclusion to change it. If it doesn't matter, don't change it. It will always bother you in the long run.

Reply:
Ok, I will revert